config/thisroot.sh

```diff
@@ -162,6 +166,14 @@ set_environment()
    else
       JUPYTER_CONFIG_DIR=$ROOTSYS/etc/notebook:$JUPYTER_CONFIG_DIR; export JUPYTER_CONFIG_DIR
    fi
+
+   if [ -z "${ROOT_INCLUDE_PATH-}" ]; then
```

Comment:
The variable name contains an extra `-` if I'm not mistaken, so this would always be zero.

Reply:
OK, I had to read up on the subtelties of `${VAR:-}` vs `${VAR-}` ...